compiler/p/codegen/FPTreeEvaluator.cpp

```diff
@@ -562,11 +562,10 @@ TR::Register *OMR::Power::TreeEvaluator::dloadEvaluator(TR::Node *node, TR::Code
    return TR::TreeEvaluator::dloadHelper(node, cg, tempReg, TR::InstOpCode::lfd);
    }
 
-
-
 TR::Register *OMR::Power::TreeEvaluator::vsplatsEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
    TR::Node *child = node->getFirstChild();
+   static bool disableDirectMove = feGetEnv("TR_disableDirectMove") ? true : false;
```

Comment:
Possibly.  Or, both functions could upgrade to a true Option rather than an environment variable, which is my preference.  However, given the overwhelming precedent to use both in this codebase already, I'm fine with it.